scripts/export-framework-applicant-details.py

```diff
@@ -37,7 +37,7 @@
     client = DataAPIClient(get_api_endpoint_from_stage(STAGE), API_TOKEN)
     now = datetime.datetime.now()
 
-    filename = OUTPUT_DIR + os.sep if OUTPUT_DIR else ""
+    filename = (OUTPUT_DIR if OUTPUT_DIR else ".") + os.sep
```

Comment:
I've made it require a directory - makes sense.  I was just thoughtlessly fixing bad behaviour when I made that bugfix for current directory.